Dashboard/app/js/lib/models/models.js

```diff
@@ -389,7 +389,7 @@ FLOW.QuestionAnswer = FLOW.BaseModel.extend({
   collectionDate: DS.attr('number'),
   surveyInstanceId: DS.attr('number'),
   iteration: DS.attr('number'),
-  questionID: DS.attr('string'),
+  questionID: DS.attr('string'), //TODO should be number?
```

Comment:
Should be a number but is quite widely used so better we tackle in a separate issue.

Reply:
That's why I made it a TODO, and not simply changed it :)

Reply:
See #2516.